public/apps/configuration/utils/audit-logging-view-utils.tsx

```diff
@@ -215,3 +194,9 @@ export function renderComplianceSettings(complianceSettings: ComplianceSettings)
     </>
   );
 }
+
+export async function updateAuditLogging(http: HttpStart, updateObject: AuditLoggingSettings) {
+  return await http.put(`${API_ENDPOINT_AUDITLOGGING_UPDATE}`, {
```

Comment:
Seems no need to use string literal here.

Reply:
Error shows if removed.